model-optimizer/setup.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
If I am not mistaken we should have an empty line before and after `http://www.apache.org/licenses/LICENSE-2.0`

Reply:
Sure. Just copying issue.